### PR TITLE
FM2-449: Change the dependency from PowerMock to Mockito in Controller test

### PR DESCRIPTION
**Pull request by theanandankit**
Issue Link: https://issues.openmrs.org/browse/FM2-449

### CODE REVIEW

**Comment by theanandankit** [comment]
@ibacher PTAL

File: pom.xml [hunk 1]
```diff
@@ -200,7 +200,13 @@
...
 				<groupId>org.mockito</groupId>
 				<artifactId>mockito-core</artifactId>
-				<version>3.2.4</version>
+				<version>4.2.0</version>
+				<scope>test</scope>
+			</dependency>
+			<dependency>
+				<groupId>org.mockito</groupId>
+				<artifactId>mockito-inline</artifactId>
+				<version>3.9.0</version>
```
Comment (HerbertYiga) [inline pom.xml]: why not use  mockito-inline version 4.2.0 ?

Comment (theanandankit) [reply pom.xml]: Yes we can, but it's also working fine with 3.9.0.

Comment (HerbertYiga) [reply pom.xml]: > Yes we can, but it's also working fine with 3.9.0.

why not use the current version!!

Comment (theanandankit) [reply pom.xml]: @HerbertYiga done the change. Thanks

**Review by ibacher — APPROVED** [review]
Beautiful! Thanks @theanandankit!

**Pull request merged**